DCS_STATUS/StatusVmon904HV.py

```diff
@@ -0,0 +1,1281 @@
+#!/usr/bin/env python
+
+import cx_Oracle
+import ROOT
+import os
+import time
+from datetime import datetime
+from operator import itemgetter
+from array import array
+import argparse
+from argparse import RawTextHelpFormatter
+
+#argparse
+parser = argparse.ArgumentParser(description='''Retrieve from the database the vmon, imon and status informations \nfor qc8 and create a root file for the asked chambers. \nTo execute the code just type \n\npython StatusVmon904HV.py \n\nand then insert the Start date, the End date of the monitor scan, \nthe number of chambers to look and the position of chmabers in the \nqc8 stand \n\n guide: https://twiki.cern.ch/twiki/bin/view/CMS/QC8MonitoringHVLV''', formatter_class=RawTextHelpFormatter)
+
+args = parser.parse_args()
+#print args.accumulate(args.integers)
+
+#import DB credentials
+from gempython.utils.wrappers import envCheck
+envCheck("GEM_ONLINE_DB_NAME")
+envCheck("GEM_ONLINE_DB_ACCOUNT")
+
+dbName = os.getenv("GEM_ONLINE_DB_NAME")
+dbAccount = os.getenv("GEM_ONLINE_DB_ACCOUNT")
+
+#in the DB the DeltaV between pins are saved, not the V from ground
+
+#time interval considered to extract the data
+#sta_period = "'2018-07-17 08:00:01'"
+#end_period = "'2018-07-17 14:30:01'"
+
+sta_period = raw_input("Insert UTC start time in format YYYY-MM-DD HH:mm:ss\n")
+type(sta_period)
+end_period = raw_input("Insert UTC end time in format YYYY-MM-DD HH:mm:ss\n")
+type(end_period)
+
+start=sta_period.replace(" ", "_")
+end=end_period.replace(" ", "_")
+start=start.replace(":", "-")
+end=end.replace(":", "-")
+
+#add ' at beginning and end to have the date in the format for the query
+sta_period = "'" + sta_period + "'"
+end_period = "'" + end_period + "'"
+
+fileName = "QC8_HV_monitor_UTC_start_"+start+"_end_"+end+".root"
+f1=ROOT.TFile( fileName,"RECREATE")
+
+#divide the monitoring period in three periods 
+#the limits between period 1 and 2 is defined by the first  date in which we changed the mapping of HV: 03-Apr-2019 13.10
+#the limits between period 2 and 3 is defined by the second date in which we changed the mapping of HV: 15-Apr-2019 08:15
+
+#in the second date only the LV mapping is changed, but the datapoints change. So we define three periods even if we have
+#only two mappings
+
+mappingChangeDate = []
+firstMappingChange 	= datetime( 2019, 04, 03, 13, 10, 00 )
+secondMappingChange 	= datetime( 2019, 04, 15,  8, 15, 00 ) 
+mappingChangeDate.append( firstMappingChange )
+mappingChangeDate.append( secondMappingChange )
```

Comment:
I would place it in a separate module and load here. Thus, if we have new mapping change in the future (e.g. for GE2/1 or ME0) it will be easier to find a place to change